api/managementpb/dbaas/psmdb_cluster.proto

```diff
@@ -0,0 +1,213 @@
+syntax = "proto3";
+
+package dbaas.v1beta1;
+
+option go_package = "api/managementpb/dbaas;dbaasv1beta1";
+
+import "github.com/mwitkow/go-proto-validators/validator.proto";
+import "google/api/annotations.proto";
+import "managementpb/dbaas/dbaas.proto";
+
+// PSMDBClusterState represents PSMDB cluster CR state.
```

Comment:
Please add TODO with a ticket number that we discussed about one extra state there